### DISCUSSION TITLE
.deb, .rpm packaging

### DISCUSSION THREAD

**Discussion by dataselfservice** [grafana/grafana#39493] [Platform]
Hi, 

there is some relevant ones open (e.g. https://github.com/grafana/grafana/issues/30963). After 8.0.5, it appears .deb and .deb packaging does not work. 

Requesting to add relevant documentation and restore `make pkg-deb pkg-rmp`, or indicate a better supported alternative. 

Regards,
Andrea

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
